rust-tests/cbmc-reg/CodegenStatic/main.rs

```diff
@@ -3,4 +3,5 @@
 static STATIC: [&str; 1] = ["FOO"];
 fn main() {
     let _x = STATIC[0];
+    assert!(_x.as_bytes()[0] == b'F');
```

Comment:
A few suggestions:
1. Name it `x` now since it's being used
2. Store the result of `x.as_bytes` in another variable
3. Then add two more assertions for byte positions 1 and 2